external/asf4-drivers/driver_init.c

```diff
@@ -290,6 +290,31 @@ void system_init(void)
     _is_initialized = true;
 }
 
+void base_init(void)
```

Comment:
does this refer to bitbox base? i would spell it would everywhere tbh, `base` is so generic that it is easy to mistake it for something else.

Reply:
ack